code/game/objects/items/crayons.dm

```diff
@@ -78,8 +78,14 @@
 					return
 				to_chat(user, "You start drawing a letter on the [target.name].")
 			if("graffiti")
+				drawtype = input("Choose the graffiti.", "Crayon scribbles") in list("amyjon","face","matt","revolution","engie","guy","end","dwarf","uboa")
+				if(get_dist(target, user) > 1 || !(user.z == target.z))

```

Comment:
Should probably have a `|| !drawtype` here so it will also return if no drawtype is input, such as by someone crashing.